x-pack/plugins/lens/public/embeddable/embeddable_component.tsx

```diff
@@ -52,7 +52,8 @@ export type TypedLensByValueInput = Omit<LensByValueInput, 'attributes'> & {
     | LensAttributes<'lnsDatatable', DatatableVisualizationState>
     | LensAttributes<'lnsMetric', MetricState>
     | LensAttributes<'lnsHeatmap', HeatmapVisualizationState>
-    | LensAttributes<'lnsGauge', GaugeVisualizationState>;
+    | LensAttributes<'lnsGauge', GaugeVisualizationState>
+    | LensAttributes<string, unknown>;
```

Comment:
Ideally it would be nice to have a way to make the developer express a stronger type definition for its own plugin. But that can be done in a follow up (will open an issue for that)